### PR TITLE
Create integration tests for v1 payload for user info

### PR DESCRIPTION
**Pull request by bidetofevil**
## Goal

Integration tests for v1 session payload for user info. Need to switch to v2 when that is hooked up.

### CODE REVIEW

**Comment by bidetofevil** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/705?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#705** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/705?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#704** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/704?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bidetofevil and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/features/UserFeaturesTest.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            IntegrationTestRule.newHarness(startImmediately = false)
+        }
+    )
+
+    @Test
+    fun `user info setting and clearing`() {
+        with(testRule) {
+            harness.androidServicesModule.preferencesService.userIdentifier = "customId"
+            harness.androidServicesModule.preferencesService.username = "customUserName"
+            harness.androidServicesModule.preferencesService.userEmailAddress = "custom@domain.com"
```
Comment (fractalwrench) [inline embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/features/UserFeaturesTest.kt:28]: nit: minor style preference for extracting a local variable of `preferencesService` here, or using `.apply {}`

**Review by fractalwrench — APPROVED** [review]
LGTM

File: embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/features/UserFeaturesTest.kt (hunk 1 above)
Comment (bidetofevil) [reply embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/features/UserFeaturesTest.kt:28]: cool will do

**Pull request merged**